pyiron_atomistics/lammps/base.py

```diff
@@ -459,9 +459,11 @@ def collect_output(self):
             ),
             group_name="output",
         )
-
-        if len(self.output.cells) > 0:
-            final_structure = self.get_structure(iteration_step=-1)
+        if len(hdf_dict["output/generic/cells"]) > 0:
+            final_structure = self.structure.copy()
+            final_structure.indices = hdf_dict["output/generic/indices"][-1]
+            final_structure.positions = hdf_dict["output/generic/positions"][-1]
+            final_structure.cell = hdf_dict["output/generic/cells"][-1]
             if final_structure is not None:
```

Comment:
Can never be `None`, so if check is unnecessary.

Reply:
I removed the if statement